runtime/rococo/src/lib.rs

```diff
@@ -1549,6 +1549,8 @@ pub mod migrations {
 	pub type Unreleased = (
 		pallet_society::migrations::VersionCheckedMigrateToV2<Runtime, (), ()>,
 		pallet_im_online::migration::v1::Migration<Runtime>,
+		/* Asynchronous backing migration */
+		parachains_scheduler::migration::v1::MigrateToV1<Runtime>,
```

Comment:
The pre/post hooks of the `parachains_scheduler::migration::v1::MigrateToV1<Runtime>` migrations are failing on Rococo and Westend and the `on_runtime_upgrade`s are noops due to onchain version != 0. 

Do these migrations need to execute? I can look into it if you like.

Reply:
They were probably failing because the migration was duplicated after the recent merge of the master branch. Thank you for the heads up